Cheaper Effort/Serivces/NewRecipeService.cs

```diff
@@ -17,15 +17,21 @@ public NewRecipeService(ProjectDbContext context)
 
         public async Task addRecipeToDBAsync(Recipe Recipe, SelectList Ingredients, string[] ingredientIds)
         {
+            int points = 0;
+            
             Guid id = Guid.NewGuid();
 
             Recipe.Id = id;
 
             await _context.Recipes.AddAsync(Recipe);
             await _context.SaveChangesAsync();
 
+            points = points + ((int)(Math.Round(Recipe.Time)) * 20) + (Recipe.Difficult_steps * 30);
```

Comment:
make a separate function to count points so that we can test it

Reply:
Will do with the next pull request :)